### ISSUE TITLE
Different result with same input

### ISSUE THREAD

**Issue by ranaraumik16** [lovasoa/salesman.js#8]
I got different result with same input points.
Correct Result

![salesmen right solution](https://user-images.githubusercontent.com/65937635/153998614-aa50b2dc-79c4-4e66-b13f-7ee4b84ab350.JPG)
![salesmen right solution plot](https://user-images.githubusercontent.com/65937635/153998619-11d2f486-40b3-4a6e-9faf-36bfdb7a9f93.JPG)

Incorrect result
![salesmen wrong solution](https://user-images.githubusercontent.com/65937635/153998643-0d625086-df49-4526-a0f7-eb377ef4f81d.JPG)
![salesmen wrong solution plot](https://user-images.githubusercontent.com/65937635/153998651-ac2176aa-6eb1-4877-a489-6c6cce77d1f5.JPG)

**Comment by lovasoa** [comment]
Yes, it uses simulated annealing, so the longer it runs, the better the results. Finding the best solution whatever the input is np-hard
